lib/nerves_network/dhclientv4_conf.ex

```diff
@@ -147,21 +186,22 @@ defmodule Nerves.Network.Dhclientv4Conf do
     interface "<%= @interface %>" {\n\
     <%= if @host_name do %>  send host-name "<%= @host_name %>";\n<% end %>\
     <%= if @vendor_class_identifier do %>  send vendor-class-identifier "<%= @vendor_class_identifier %>";\n<% end %>\
+    <%= if @hardware_type do %>\
+    <%= if @client_identifier do %>  send dhcp-client-identifier <%= @client_identifier %>;\n<% end %>\
+    <% else %>\
     <%= if @client_identifier do %>  send dhcp-client-identifier "<%= @client_identifier %>";\n<% end %>\
+    <% end %>\
```

Comment:
I struggled with extra newlines in other EEx templates.  That is awesome you figured out the "\" trick

Reply:
:)

Reply:
'twas drivin' me nuts :)